README.md

```diff
@@ -126,6 +126,8 @@ Ollama bundles model weights, configuration, and data into a single package, def
 
 ## Building
 
+You will also need a C/C++ compiler such as GCC for MacOS and Linux or Mingw-w64 GCC for Windows.
```

Comment:
I feel like this should be in the development.md with the other build steps, but just my opinion

Reply:
I think it's ok to put the full list of reqs here to go from zero to built (incl. adding cmake, etc).

Reply:
I agree with @BruceMacD on this. There's duplication between README.md and docs/development.md. This is surfacing the content in docs/development.md to README.md because there's been multiple instances of users trying to build from source and not knowing the C/C++ requirements.